### PR TITLE
add data credit usage map layer

### PR DESCRIPTION
**Pull request by allenan**

### CODE REVIEW

File: components/InfoBox/HexDetailsInfoBox.js [hunk 1]
```diff
@@ -44,8 +61,12 @@ const HexDetailsInfoBox = () => {
         path: `/cities/${hotspot.geocode.cityId}`,
         title: formatLocation(hotspot.geocode),
       },
+      {
+        icon: <img src="/images/dc.svg" />,
+        title: `${mapHex?.dc?.toLocaleString() || 0} DC (30d)`,
```
Comment (allenan) [inline components/InfoBox/HexDetailsInfoBox.js]: note that this isn't actually 30d, it was taking too long to backfill using my current strategy of walking blocks. I think we may need a better API for this

**Review by danielcolinjames — APPROVED** [review]
looks awesome! I was thinking it would benefit from a scale / legend, but I realized the one I made for the transmit scale wasn't very well componentized so that would fit better in its own PR I think

File: components/Icons/Hex.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const Hex = ({ className }) => {
```
Comment (danielcolinjames) [inline components/Icons/Hex.js:1]: 🔥

File: components/Map/MapControls.js [hunk 3]
```diff
@@ -72,6 +72,36 @@ const MapControls = () => {
...
+          {
+            'opacity-100 pointer-events-auto':
+              mapLayer === 'dc' && !showMapLayers,
+            'opacity-0 pointer-events-none translate-y-10':
+              mapLayer !== 'dc' || showMapLayers,
+          },
+        )}
+      >
+        <div className="rounded-lg flex flex-row items-center justify-between titlebox-blur">
+          <a
```
Comment (danielcolinjames) [inline components/Map/MapControls.js:87]: maybe a DC docs link? or no link for now would work too

**Review by danielcolinjames — APPROVED** [review]
looks good! noticed there's a link to the docs about transmit scales on the title of the legend, but other than that looks good to merge 👍

File: components/Map/MapControls.js (hunk 3 above)
Comment (allenan) [reply components/Map/MapControls.js:87]: good catch, updated

**Pull request merged**